setup/openchange_neworganization

```diff
@@ -67,6 +71,8 @@ if opts.deprovision:
     openchange.deprovision_organization(setup_path, provisionnames, lp, creds)
     openchange.unregister(setup_path, provisionnames, lp, creds)
 else:
-    if openchange.provision_organization(setup_path, provisionnames, lp, creds):
-        openchange.register(setup_path, provisionnames, lp, creds)
+    if openchange.provision_organization(setup_path, provisionnames, lp, creds, ignore_already_exists=opts.ignore_already_exists):
```

Comment:
This script has right now only one function, provision an organization, so everytime I tried to enable all groups it will try to provision, again, the organization